build/scripts/create-perftests.ps1

```diff
@@ -0,0 +1,32 @@
+# Create the PerfTests directory under Binaries\$(Configuration).  There are still a number
```

Comment:
There are both dependencies in DevDivInesrtionFiles.sln and inside roslyn-internal scripts.